Samples/JExtractJNISampleApp/src/main/java/com/example/swift/HelloJava2SwiftJNI.java

```diff
@@ -40,8 +41,10 @@ static void examples() {
 
         MySwiftClass.method();
 
-        MySwiftClass myClass = MySwiftClass.init(10, 5);
-        MySwiftClass myClass2 = MySwiftClass.init();
+        try (var arena = new ConfinedSwiftMemorySession(Thread.currentThread())) {
```

Comment:
Nitpick that this may want to be `SwiftArena.ofConfined()`

Reply:
agreed, but the problem is that Java 7 does not support static methods on interfaces.

Reply:
At least we can make a `()` constructor that gets the current thread as well then, we can do this in a follow up